### DISCUSSION TITLE
v0.11.0(重要重要重要)

### DISCUSSION THREAD

**Discussion by lanyeeee** [lanyeeee/bilibili-manga-downloader#67] [Announcements]
`portable`为免安装版，解压后双击直接运行，容易被杀毒软件(如Windows Defender)误判为病毒

[关于被杀毒软件误判为病毒](https://github.com/lanyeeee/bilibili-manga-downloader?tab=readme-ov-file#%E5%85%B3%E4%BA%8E%E8%A2%AB%E6%9D%80%E6%AF%92%E8%BD%AF%E4%BB%B6%E8%AF%AF%E5%88%A4%E4%B8%BA%E7%97%85%E6%AF%92)

### 新增

- 优化防封号逻辑 #66 

### 完整的Cookie非常长，注意复制完整

![image](https://github.com/user-attachments/assets/48918529-a685-449e-8aec-5bfeb3beb313)

<hr /><em>This discussion was created from the release <a href='https://github.com/lanyeeee/bilibili-manga-downloader/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by 274509635** [comment]
加蜜罐参数了。
建议还是别搞了，他今天改一下，明天改一下，参数不一致就封。
我都损失2个账号了，好害人。
不敢玩了。

**Reply by lanyeeee** [reply]
有蜜罐参数的证据吗？

**Reply by 274509635** [reply]
我看请求中，无脑加了一个参数，没有这些参数的都遭了

**Reply by huamanfei** [reply]
你申请解封就行，我的解了。我喷他iPad版好几年不能出个横版，还不让用户自己下载看提高体验。反正就这个点喷了一百来字，解封了。

**Comment by lkjmnbvcx11** [comment]
只有他的官方途径不会触发，感觉只能模拟成通过官方途径网页浏览那种模式扒图了，我感觉集云数据这个网站就是这么扒的，像看漫画一样，边“看”边扒。

**Comment by ethicfan** [comment]
更新了以后反而下不了了。给的错误提示是——0:获取lmageToken失败，预料之外的code:BiliResp{code: 98, msg:"请求失效，请刷新页面重试"，data: None}
按照我用英语硬理解，感觉是没有券连接不上？但我都是买了章回的啊，而且据说软件能内个啥不是（捂脸）

**Comment by lanyeeee** [comment]
先观望一段时间了，现在那边更新太频繁了，即使解决了也是能用两三天然后继续封

**Comment by lkjmnbvcx11** [comment]
也是，大佬先等等再说，也不急在一时

**Comment by lanyeeee** [comment]
写了个简陋的脚本，勉强能用，目前没发现封号，不知道能用多久
https://github.com/lanyeeee/bilibili-manga-download-script

**Reply by lanyeeee** [reply]
哈哈，才过去半天就不能用了，现在秒封。
周末也上班吗，这么拼

**Reply by lkjmnbvcx11** [reply]
现在我都怀疑是不是论坛里面进了内鬼了，不然大佬你更新一个，那边就有相应的措施对抗一个

**Comment by lanyeeee** [comment]
[v0.2.0能正常使用了，但是我不发](https://github.com/lanyeeee/bilibili-manga-download-script/issues/4)

**Reply by lkjmnbvcx11** [reply]
这阵子还是先苟住吧
